test/functional/ui/wildmode_spec.lua

```diff
@@ -31,6 +31,21 @@ describe("'wildmenu'", function()
     ]])
   end)
 
+  it(':sign <tab> <space> hides wildmenu #8453', function()
+    command('set wildmode=full')
+    -- only a regression if status-line open
+    command('set laststatus=2')
+    command('set wildmenu')
+    feed(':sign <tab> ')
+    screen:expect([[
+                               |
+      ~                        |
+      ~                        |
+      [No Name]                |
+      :sign define ^            |
+    ]])
```

Comment:
To add more rigor to the test, let's add an extra `screen:expect()` before the `<space>` is input:

```lua
     feed(':sign <tab>')
     screen:expect([[
                                |
       ~                        |
       ~                        |
       define  jump  list  >    |
       :sign define^             |
     ]])
     feed('<space>')
     screen:expect([[
                                |
       ~                        |
       ~                        |
       [No Name]                |
       :sign define ^            |
     ]])
```

After that LGTM! I tested the patch locally and didn't find any new regressions.

Reply:
Awesome! I'll make the change--thank you for the guidance!